internal/metastore/catalog.go

```diff
@@ -135,6 +135,10 @@ type DataCoordCatalog interface {
 	AlterSegmentIndexes(ctx context.Context, newSegIdxes []*model.SegmentIndex) error
 	DropSegmentIndex(ctx context.Context, collID, partID, segID, buildID typeutil.UniqueID) error
 
+	SaveImportTask(task any) error
```

Comment:
splitting it into two separate functions.
Please pass the specific object that needs to be serialized to this function, rather than passing a task interface.